mongo/integration/gridfs_test.go

```diff
@@ -29,6 +30,83 @@ func TestGridFS(x *testing.T) {
 	mt := mtest.New(x, noClientOpts)
 	defer mt.Close()
 
+	mt.Run("skipping download", func(mt *mtest.T) {
+		data := []byte("abc.def.ghi")
+		var chunkSize int32 = 4
+
+		testcases := []struct {
+			name string
+
+			read              int
+			skip              int64
+			expectedSkipN     int64
+			expectedSkipErr   error
+			expectedRemaining string
```

Comment:
This is fine, but it would require us to change every test if the `data` variable changed. Do you think it'd be more robust to specify the number of bytes remaining instead? e.g. If the test skips no bytes, the total remaining is `len(data)` and if it skips all bytes, the remaining is `0`.

Reply:
Good idea, done.